### DISCUSSION TITLE
can I use shell in toolchain.ini?

### DISCUSSION THREAD

**Discussion by stsp** [mesonbuild/meson#12781] [Q&A] [answered]
I use `--cross-file toolchain.ini`.
Can I use shell substitutions in that
file to evaluate paths with pkg-config?

**Comment by dcbaker** [comment, marked as answer]
No, machine files do not allow shell evaluation. They do have a some basic concatenation, math, and interpolation abilities, but nothing like a turing completely shell language. If you really need that you'll need to approach it the other way and generate the toolchain file from a script and then consume that.

**Reply by stsp** [reply]
Thanks, generating works.
Hope in the future the shell substitutions
will be supported.
